encrypter.cpp

```diff
@@ -0,0 +1,174 @@
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <ios>
+#include <random>
+#include <sstream>
+#include <utility>
+#include <iterator>
+
+using Byte = char;
+
+//=====put_error=====//
+void put_error(const std::string& error_message)
+{
+  std::cout << error_message;
+  exit(1);
+}
+
+//=====shuffle=====//
+void shuffle(std::vector<Byte>& input)
+{
+  std::default_random_engine gen;
+
+  for (std::vector<Byte>::iterator iter = input.begin(); iter < input.end(); ++iter)
```

Comment:
It going to be easer to use indexes here instead of iterators. iterators for sequential access but you access vector randomly anyway.